bundles/org.openhab.binding.bluetooth.daikinmadoka/src/main/java/org/openhab/binding/bluetooth/daikinmadoka/handler/DaikinMadokaHandler.java

```diff
@@ -290,8 +335,19 @@ public void handleCommand(ChannelUID channelUID, Command command) {
         }
     }
 
+    /**
+     * 2 actions need to be done: disable the notification AND reset the filter timer
+     */
+    private void resetCleanFilterIndicator() {
+        logger.debug("[{}] resetCleanFilterIndicator()", super.thing.getUID().getId());
+        submitCommand(new DisableCleanFilterIndicatorCommand());
+        submitCommand(new ResetCleanFilterTimerCommand());
+    }
+
     @Override
     public void onCharacteristicUpdate(BluetoothCharacteristic characteristic) {
+        logger.debug("[{}] onCharacteristicUpdate({})", super.thing.getUID().getId(),
+                HexUtils.bytesToHex(characteristic.getByteValue()));
```

Comment:
Because `byteToHex` can be an expensive(ish) operation please wrap this in a logging level check.
```suggestion
        if(logger.isDebugEnabled()){
            logger.debug("[{}] onCharacteristicUpdate({})", super.thing.getUID().getId(),
                HexUtils.bytesToHex(characteristic.getByteValue()));
        }
```